src/main/java/Task.java

```diff
@@ -0,0 +1,112 @@
+public class Task {
```

Comment:
Could you consider adding header comments to the classes? Saw this problem in some other classes too.